tools/run_local_ironic.sh

```diff
@@ -170,10 +170,15 @@ fi
 
 BASIC_AUTH_MOUNTS=""
 IRONIC_HTPASSWD=""
```

Comment:
I wonder if we need this as we don't use it undefined anymore

Reply:
Fixed